keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/binary/MaxVertex.java

```diff
@@ -11,7 +13,9 @@
      * @param left  one of the vertices to find the maximum of
      * @param right one of the vertices to find the maximum of
      */
-    public MaxVertex(DoubleVertex left, DoubleVertex right) {
+    @ExportVertexToPythonBindings
+    public MaxVertex(@LoadParentVertex(THN_NAME) DoubleVertex left,
```

Comment:
THN_NAME?

Reply:
Do we need left and right getters on this?

Reply:
Sorted the naming.  We don't really need the getters here (and can't really put them on without adding extra duplication at the moment.